helium/src/main/groovy/com/stanfy/helium/internal/utils/Names.java

```diff
@@ -41,7 +41,7 @@ public static String prettifiedName(final String s) {
     StringBuilder result = new StringBuilder(s);
     for (int i = 0; i < result.length(); i++) {
       boolean shouldCap = false;
-      while (i < result.length() && (result.charAt(i) == '_' || result.charAt(i) == '-')) {
+      while (i < result.length() && (result.charAt(i) == '_' || result.charAt(i) == '-' || result.charAt(i) == ' ')) {
```

Comment:
- Add it to the method test please.
- We usually do something like `prettifiedName(canonicalName(s))`, where `canonicalName handles spaces`
- Also,  `Type`s and `Service`s provide (IIRC) `getCanonicalName()` method